src/components/filterModal/statsList/StatsList.vue

```diff
@@ -90,6 +92,52 @@ export default {
                 }
             });
         },
+        hasFiltered(data) {
+            if (this.searchQuery) {
+                let array;
+                if (Array.isArray(data.data)) {
+                    array = data.data;
+                } else if (data.data.variants) {
+                    // INHERITANCE - custom handler
+                    array = data.data.variants.concat(data.data.family);
+                }
+                const result = array.filter(item => (Array.isArray(item) ? item[0].toLowerCase()
```

Comment:
Next time you can write it shorter with &&, like: 
Array.isArray(item) && item[0].toLowerCase().includes(this.searchQuery.toLowerCase())